examples/DocSum/deploy/gaudi.md

```diff
@@ -0,0 +1,485 @@
+  -H 'Content-Type: application/json'
+```
+
+The output is the summary of the input given to this microservice
+
+### Whisper Microservice
+
+```bash
+ curl http://${host_ip}:7066/v1/asr \
+     -X POST \
+     -d '{"audio":"UklGRigAAABXQVZFZm10IBIAAAABAAEARKwAAIhYAQACABAAAABkYXRhAgAAAAEA"}' \
+     -H 'Content-Type: application/json'
+```
+
+Here is the output:
+```
+  {"asr_result":"you"}
+```
+
+### MegaService
+
+You can upload documents(.txt,.doc,.pdf), audio and video and get the summarization of the content.
+
+::::::{tab-set}
+:::::{tab-item} Text
+:sync: Text
+
+The megaservice accepts intput files in txt, pdf, doc format or a plain text in the message parameter.
+```bash
+curl http://${host_ip}:8888/v1/docsum \
+    -H "Content-Type: multipart/form-data" \
+    -F "type=text" \
+    -F "messages=Text Embeddings Inference (TEI) is a toolkit for deploying and serving open source text embeddings and sequence classification models. TEI enables high-performance extraction for the most popular models, including FlagEmbedding, Ember, GTE and E5." \
+    -F "max_tokens=32" \
+    -F "language=en" \
+    -F "stream=true"
+```
+
+The output will be the summarization of the text content. We can also upload files and modify the other parametes such as the streaming mode and language.
+
+```bash
+curl http://${host_ip}:8888/v1/docsum \
+   -H "Content-Type: multipart/form-data" \
+   -F "type=text" \
+   -F "messages=" \
+   -F "files=@/path to your file (.txt, .docx, .pdf)" \
+   -F "max_tokens=32" \
+   -F "language=en" \
+   -F "stream=true"
+```
+:::::
+
+:::::{tab-item} Audio
+:sync: Audio
+
+Audio uploads are not supported through curl command, use the UI to upload it. You can pass base64 string of the audio file as follows :
+
+```bash
+curl http://${host_ip}:8888/v1/docsum \
+   -H "Content-Type: multipart/form-data" \
+   -F "type=audio" \
+   -F "messages=UklGRigAAABXQVZFZm10IBIAAAABAAEARKwAAIhYAQACABAAAABkYXRhAgAAAAEA" \
+   -F "max_tokens=32" \
+   -F "language=en" \
+   -F "stream=true"
+```
+:::::
+
+:::::{tab-item} Video
+:sync: Video
+
+Video uploads are not supported through curl command, use the UI to upload it. You can pass base64 string of the video file as the value for message parameter as shown here :
+
+```bash
+curl http://${host_ip}:8888/v1/docsum \
+   -H "Content-Type: multipart/form-data" \
+   -F "type=video" \
+   -F "messages=convert your video to base64 data type" \
+   -F "max_tokens=32" \
+   -F "language=en" \
+   -F "stream=true"
+
+```
+:::::
+
+::::::
+
+When dealing with longer context of the content to be summarized, we can use different summarization stratergies such as auto, stuff, truncate, map_reduce or refine. Depending on various factors like models context size and input tokens we can select the stratergy that best fits.
+
+1. Auto : In this mode we will check input token length, if it exceed MAX_INPUT_TOKENS, summary_type will automatically be set to refine mode, otherwise will be set to stuff mode.
```

Comment:
```suggestion
1. Auto : In this mode we will check input token length, if it exceeds MAX_INPUT_TOKENS, the summary_type will automatically be set to refine mode, otherwise will be set to stuff mode.
```